### PR TITLE
Introduce setup.py

### PR DESCRIPTION
**Pull request by jmuk**

### CODE REVIEW

**Comment by jmuk** [comment]
cc: @tbetbetbe

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+import setuptools
+
+from pip.req import parse_requirements
+
+requirements = [
+    'oslo.utils>=3.4.0',
+    'pyyaml>=3.11',
+    'taskflow>=1.25.0',
+    'yapf>=0.6.2'
```
Comment (ethanbao) [inline setup.py]: Just realize that PR16 (https://github.com/googleapis/artman/pull/16/files) hasn't been merged. Can you add the three extra requirements listed in that PR into this one?

Comment (jmuk) [inline setup.py]: done

**Comment by ethanbao** [comment]
LGTM

**Pull request merged**